src/app/core/services/datacenter.ts

```diff
@@ -113,4 +115,29 @@ export class DatacenterService {
     const url = `${this._newRestRoot}/seeds/${seedName}/settings`;
     return this._httpClient.get<SeedSettings>(url);
   }
+
+  // only admins can call following endpoints
+  get adminSeeds(): Observable<AdminSeed[]> {
+    if (!this._adminSeeds$) {
+      this._adminSeeds$ = merge(this._adminSeedsRefresh$, this._refreshTimer$)
+        .pipe(switchMap(_ => this._getAdminSeeds()))
+        .pipe(shareReplay({refCount: true, bufferSize: 1}));
+    }
+
+    return this._adminSeeds$;
+  }
+
+  private _getAdminSeeds(): Observable<AdminSeed[]> {
+    const url = `${this._restRoot}/admin/seeds`;
```

Comment:
Is this a new endpoint? How does it differ from the original seeds endpoint?

Reply:
We're now having
1. `seeds` via `/api/v1/seed` -> used for every user group and only returns an array of seed names
2. `seedSettings` via `/api/v2/seeds/{seedName}/settings` -> returns some settings (e.g. mla) that are needed in wizard and therefor need to be readable for all user groups
3. "new" `adminSeeds` via `/api/v1/admin/seeds` -> returns all seeds including its whole settings (and also the backupRestore values). This ep is only available for admins, as it includes configurations like e.g. proxy-settings.